packages/backend-core/src/security/auth.ts

```diff
@@ -0,0 +1,29 @@
+import { env } from ".."
+
+export const PASSWORD_MIN_LENGTH = 8
+export const PASSWORD_MAX_LENGTH = 512
+
+export function validatePassword(
+  password: string
+): { valid: true } | { valid: false; error: string } {
+  if (env.isDev() && !env.isTest() && password) {
+    // We accept any password while on development
+    return { valid: true }
+  }
```

Comment:
This absolutely terrifies me. If this ever makes it to production by mistake it would be catastrophic.

Reply:
Same, I just cleaned it :)